src/main/java/com/epam/spring/homework1/other/OtherBeanB.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.spring.homework1.other;
+
+import com.epam.spring.homework1.beans.BeanB;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OtherBeanB {
+    private BeanB beanB;
+
+@Autowired
+    public void setBeanB(BeanB beanB){
+    this.beanB = beanB;
+    System.out.println(this.getClass().getSimpleName()+ ". " + beanB.getClass().getSimpleName()+ " was injected through the setter");
+}
```

Comment:
nit: try to keep code style and indents consistent
There are a lot of different conventions, but you can just install google-java-format plugin to your IDE or add it as a Maven plugin, that would be totally enough